src/transformers/__init__.py

```diff
@@ -401,6 +401,12 @@
         "TrOCRConfig",
         "TrOCRProcessor",
     ],
+    "models.tvlt": [
+        "TVLT_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "TvltConfig",
+        "TvltAudioFeatureExtractor",
```

Comment:
```suggestion
        "TvltFeatureExtractor",
```
Please just use the term `TvltFeatureExtractor`